test/modules/map/components/layerManager/LayerManager.test.js

```diff
@@ -67,6 +67,27 @@ describe('LayerManager', () => {
 			expect(toggleElement.checked).toBeFalse();
 		});
 
+		it('with one hidden of two layer displays one layer item', async () => {
```

Comment:
Just as suggestions for a better description:
`it('displays one out of two layers - one is hidden'`
or
`it('takes hidden layers into account'`